Formula/rainbarf.rb

```diff
@@ -3,8 +3,7 @@ class Rainbarf < Formula
   homepage "https://github.com/creaktive/rainbarf"
   url "https://github.com/creaktive/rainbarf/archive/v1.4.tar.gz"
   sha256 "066579c0805616075c49c705d1431fb4b7c94a08ef2b27dd8846bd3569a188a4"
-  # license ["Artistic-1.0", "GPL-1.0"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "Artistic-1.0"
+  license any_of: ["Artistic-1.0", "GPL-1.0"]
```

Comment:
```suggestion
  license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
```

Matches the license of [`perl`](https://github.com/Homebrew/homebrew-core/blob/b6af2713c9e0381fc21dcac3d97207287f3362b8/Formula/perl.rb#L6)